### ISSUE TITLE
Tables constructors are squished into a single line

### ISSUE THREAD

**Issue by willmoffat** [trixnz/lua-fmt#1]
This looks like a very promising project, however it seems that table constructors are forced into a single line, making them hard to read.

Reduced testcase:
```lua
local f = { foo = true, bar = false,
    t = "It was a bright cold day in April, and the clocks were striking 13."
}
```

What luafmt gives:
```lua
local f = {foo = true, bar = false, t = "It was a bright cold day in April, and the clocks were striking 13."}
```

What I expect:
```lua
local f = {
    foo = true,
    bar = false,
    t = "It was a bright cold day in April, and the clocks were striking 13."
}
```

**Comment by trixnz** [comment]
This should be fixed (or, at the very least, improved) by v2.2.0: https://github.com/trixnz/lua-fmt/commit/7b1c9f3de3b887b7746ba4a39acd357cc716773a.

Unfortunately, I haven't got around to adding the new options to the CLI invocation script yet, so you will be stuck with the defaults for now (120 line width, double quotes).

**Comment by willmoffat** [comment]
Nice! Works for me :-)
